src/routes/blog/post/public-roadmap-announcement/+page.markdoc

```diff
@@ -0,0 +1,61 @@
+
+## Aligning expectations
+We understand that knowing when new features and fixes will be available is important for planning and integrating Appwrite into your products. The public roadmap provides clear timelines and updates on our progress, helping you better plan your development cycles, communicate them to your stakeholders, and set expectations accordingly.
+
+# How to use the roadmap
+
+## Accessing the roadmap
+You can access our public roadmap on GitHub. The roadmap is organized into different projects for each release, with issues or pull requests assigned to specific versions to indicate when we plan to complete the changes. For example, you can check out the changes planned for version 1.5.8 to see what’s coming up next.
+
+## Reporting bugs and requesting features
+To report a bug or request a new feature, simply create an issue on our GitHub repository. When submitting an issue, please provide as much detail as possible, including steps to reproduce bugs and the specific needs for feature requests. This information helps our team to understand and prioritize the issue effectively.
+
+## Engaging with the community
+The public roadmap is not just a tool for visibility but also a platform for collaboration. You can comment on existing issues, provide additional context, and even contribute to discussions around feature development. By upvoting bugs or features requests on GitHub you help us prioritize between the different issues and pull requests. Your insights and contributions are invaluable in helping us build a better Appwrite.
+
+## Staying updated
+We encourage you to check the public roadmap regularly to stay updated on our progress. You can also watch the repository to receive notifications about new issues, updates, and releases. This way, you will always be in the loop about what’s happening with Appwrite.
+
+## Flexibility and future planning
+While the new roadmap offers much more flexibility and visibility into our short-term and mid-term plans, it might still miss some bigger, longer-term projects or milestones that are in the early planning stages or going through an internal or external RFC (Request for Comments) process. These initiatives require more detailed planning and consultation before they can be added to the public roadmap.
+
+# The roadmap framework
+To ensure our focus is always on the right areas, we have developed an internal framework to guide our roadmap decisions. This framework is based on the significance of our focus areas, allowing us to set clear priorities, measure progress, understand the impact, maintain balance, and foster collaboration within the team and the Appwrite community.
+
+![Roadmap framework](/images/blog/public-roadmap-announcement/framework.png)
```

Comment:
@LauraDuRy this image was available on Figma but I didn't see it in the draft on Google Docs. Is this the right place to add it?

Reply:
The location is good 👍